### PR TITLE
Starting to work on the layout.

### PR DESCRIPTION
**Pull request by talaiziamorris**
Completely changed the basic content. Will update styles later.

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -49,7 +50,61 @@
...
+    border: 3px dotted black;
+    margin: 10%
+}
+#experience {
+    text-align: center;
+    border: 3px dotted black;
+    margin: 5%;
+    background-color: rgb(194, 165, 69);
+}
+.headings {
```
Comment (esthicodes) [inline css/index.css:68]: What does the overall design look like?

File: css/index.css [hunk 2]
```diff
@@ -49,7 +50,61 @@
...
+}
+.messages{
+    color: rgb(233, 94, 2);
+}
+.submit{
+    background-color: rgb(170, 112, 36);
+    border: .5px none;
+    margin-left: 200%;
+}
+#footer{
```
Comment (esthicodes) [inline css/index.css:99]: is there a special reason that you wrote the #footer personally designed with the border, text-align, font-family as this style?

File: css/index.css [hunk 3]
```diff
@@ -49,7 +50,61 @@
...
+    color: rgb(233, 94, 2);
+}
+.email{
+    color: rgb(233, 94, 2);  
+}
+.messages{
+    color: rgb(233, 94, 2);
+}
+.submit{
+    background-color: rgb(170, 112, 36);
```
Comment (esthicodes) [inline css/index.css:95]: did you know that the RGB color [170, 112, 36](https://convertingcolors.com/rgb-color-170_112_36.html) is a dark color, and the websafe version is hex [996600](https://convertingcolors.com/hex-color-996600.html), so a complement of this color would be [36, 94, 170](https://convertingcolors.com/rgb-color-36_94_170.html), and the grayscale version is [121, 121, 121](https://convertingcolors.com/rgb-color-121_121_121.html)? :)

**Review by esthicodes — COMMENTED** [review]
lgtm

File: css/index.css (hunk 3 above)
Comment (talaiziamorris) [reply css/index.css:95]: I didn't. Thank you!

File: css/index.css (hunk 1 above)
Comment (talaiziamorris) [reply css/index.css:68]: Still working on it. I did what I could to complete the assignment on time, but will take the time to make it more visually pleasing.

**Pull request merged**